src/liballoc_system/lib.rs

```diff
@@ -71,7 +72,49 @@ pub extern "C" fn __rust_usable_size(size: usize, align: usize) -> usize {
     imp::usable_size(size, align)
 }
 
-#[cfg(unix)]
+#[cfg(target_os = "redox")]
+mod imp {
```

Comment:
I will probably change this to be different from Unix so that it defines externs that ralloc will fulfill.